File: web_vue.client/types/viewmodel/Auth/LoginRequestViewModel.ts
// !! 此檔案由 generate-models.mjs 自動生成，請勿手動修改 !!
// 來源：Web_Vue.Server/ViewModels
/** 登入請求 DTO */
export interface LoginRequestViewModel {
  /** 使用者帳號 */
  Account: string
  /** 密碼 */
  Password: string
  /** 驗證碼 Challenge ID（由 /api/auth/captcha 取得） */
  ChallengeId: string
  /** 使用者輸入的圖形驗證碼 */
  ValidCode: string
}

File: Web_Vue.Server/ViewModels/Auth/LoginRequestViewModel.cs
namespace Web_Vue.Server.ViewModels.Auth;

/// <summary> 登入請求 DTO </summary>
public class LoginRequestViewModel
{
    /// <summary> 使用者帳號 </summary>
    public string Account { get; set; } = string.Empty;

    /// <summary> 密碼 </summary>
    public string Password { get; set; } = string.Empty;

    /// <summary> 驗證碼 Challenge ID（由 /api/auth/captcha 取得） </summary>
    public string ChallengeId { get; set; } = string.Empty;

    /// <summary> 使用者輸入的圖形驗證碼 </summary>
    public string ValidCode { get; set; } = string.Empty;
}
